.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+	"name": "Flutter.dev",
```

Comment:
I don't know much about that, .devcontainers are just a normal option these days for Dockerized environments - some VSCode devs appreciate them because it's easy to use and keeps them in the Editor. It's not required.